Formula/git-sv.rb

```diff
@@ -4,18 +4,13 @@ class GitSv < Formula
   url "https://github.com/thegeeklab/git-sv/archive/refs/tags/v1.0.5.tar.gz"
   sha256 "2688f40faa50517f8ed41a3deac93cd55f6cdd6974a3665f497ff8840bcf36d9"
   license "MIT"
-
-  head do
-    url "https://github.com/thegeeklab/git-sv.git", branch: "main"
-  end
+  head "https://github.com/thegeeklab/git-sv.git", branch: "main"
 
   livecheck do
     url :stable
     strategy :github_latest
   end
 
-  depends_on "go" => :build
```

Comment:
I think this needs to stay as the formula does a build and doesn't use existing binaries.

Reply:
Yes, removed by accident and already fixed again :)